pinot-common/src/main/java/com/linkedin/pinot/common/metadata/stream/KafkaStreamMetadata.java

```diff
@@ -43,9 +42,14 @@ public KafkaStreamMetadata(Map<String, String> streamConfigMap) {
     _zkBrokerUrl =
         streamConfigMap.get(StringUtil.join(".", Helix.DataSource.STREAM_PREFIX,
             Helix.DataSource.Realtime.Kafka.HighLevelConsumer.ZK_CONNECTION_STRING));
-    _consumerType =
-        ConsumerType.valueOf(streamConfigMap.get(StringUtil.join(".", CommonConstants.Helix.DataSource.STREAM_PREFIX,
-            CommonConstants.Helix.DataSource.Realtime.Kafka.CONSUMER_TYPE)));
+
+    String consumerTypesCsv =streamConfigMap.get(StringUtil.join(".", Helix.DataSource.STREAM_PREFIX, Helix.DataSource.Realtime.Kafka.CONSUMER_TYPE));
+    String[] parts = consumerTypesCsv.split(",");
```

Comment:
Use Guava's splitter here and use the `trimResults` option so that things like `highLevel, lowLevel` parse properly even if there's whitespace in the list.